rest/model/host.go

```diff
@@ -72,12 +81,17 @@ type DistroInfo struct {
 }
 
 type TaskInfo struct {
-	Id           *string    `json:"task_id"`
-	Name         *string    `json:"name"`
+	// Unique Identifier of this task. Can be used to fetch more informaiton about this task
```

Comment:
Do we need the second sentence? Seems like most of the other docs just say "it's the unique identifier".